### ISSUE TITLE
Issues arising from #698

### ISSUE THREAD

**Issue by jrood-nrel** [Exawind/nalu-wind#727]
I have updated the gold files to account for #698 . However, there are still problems related to:
```
what():  ABL lower boundary condition with planar averaging requires ABL statistics (enable 'boundary_layer_statistics' in input file).
```
I'm not sure if the missing mesh files seen in other tests are related. Several warnings are appearing now as well.

**Comment by sayerhs** [comment]
@jrood-nrel With c096faee4b54ff the warnings and the missing mesh files issues should be fixed. We still have 4 tests that will fail. `ablNeutralStat` uses `nccmp` and requires the solution files updated. The other three tests are failing because of solution divergence: `ablUnstableEdge`, `ablUnstableEdge_ra` and `tamsChannelElem`. `tamsChannelElem` is a regression that @jamelvin is looking into and is unrelated to #698. @mchurchf Can you see why unstable cases are failing? Most likely the input settings are off.

**Comment by sayerhs** [comment]
@lawrenceccheung The [ABL neutral statistics test](https://my.cdash.org/test/13988103) has been failing `nccmp` even after @jrood-nrel has updated the gold statistics file ever since #698 was merged. Is there anything peculiar to #698 that will cause this test to fail?

**Comment by lawrenceccheung** [comment]
@sayerhs That's funny, if the `abl_statistics.nc.gold` was updated then it should work with the latest build.  It's possible that the tolerances in `passfail` might need to be updated or something unless was unwittingly changed with the boundary layer statistics.  I'll take a look and see what's going on.

Lawrence

**Comment by lawrenceccheung** [comment]
@jrood-nrel @sayerhs It looks like [`passfail`](https://github.com/Exawind/nalu-wind/blob/master/reg_tests/test_files/ablNeutralStat/passfail) was updated to compare the `ablNeutralStat.nc` output against the gold file `ablNeutralStat.nc.gold`:
https://github.com/Exawind/nalu-wind/blob/a305b1501a6dd283b59ca5ee9b035ec73b756b2e/reg_tests/test_files/ablNeutralStat/passfail#L43

I think the `stats_output_file` in `ablNeutralStat.yaml` also needs to be updated to reflect the new name:
https://github.com/Exawind/nalu-wind/blob/a305b1501a6dd283b59ca5ee9b035ec73b756b2e/reg_tests/test_files/ablNeutralStat/ablNeutralStat.yaml#L267-L269

However, let me build and test this with the latest Nalu-Wind commits to make sure there's nothing else that needs to be fixed in terms of tolerances, etc.

Lawrence

**Comment by jrood-nrel** [comment]
Ah, sorry if I didn't do that right. My gold update script relies on consistent naming of files, so I don't have to manually update each test.

**Comment by lawrenceccheung** [comment]
@sayerhs Looks like you already fixed the `stats_output_file` in commit 048a62edbca7c4acd52502be03d2778a4d2a5d3a.  On my machine it looks like I also need to adjust the tolerances for one of the variables in `ablNeutralStat.nc`.  I haven't gotten a chance to test it on other machines at Sandia yet, but if it also fails on the official test machine, we can adjust.

**Comment by sayerhs** [comment]
@jrood-nrel will update the gold files. So don't worry about it. Thanks for the tip regarding file name mismatch
